assets/overlays/openstack-cinder/generated/standalone/controller.yaml

```diff
@@ -134,14 +133,8 @@ spec:
         - --leader-election-namespace=${NODE_NAMESPACE}
         - --v=${LOG_LEVEL}
         - --timeout=3m
-        - --feature-gates=Topology=$(ENABLE_TOPOLOGY)
         - --default-fstype=ext4
-        env:
-        - name: ENABLE_TOPOLOGY
-          valueFrom:
-            configMapKeyRef:
-              key: enable_topology
-              name: cloud-conf
+        env: []
```

Comment:
nit: omit this?.

Reply:
This is auto-generated (and generated from scratch each time rather than updated) so I'm thinking I can't.